client/post-editor/post-editor.jsx

```diff
@@ -695,19 +692,12 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Assign editor post ID to saved value (especially important when
-		// transitioning from an unsaved post to a saved one)
-		if ( post.ID !== this.props.postId ) {
-			this.props.setEditorPostId( post.ID );
-		}
+		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
+		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
Yeah, I noticed these notices as well :). I'm starting to think, we should avoid using the middleware to automatically trigger notices on request success. I had to revert my own save settings global notices here https://github.com/Automattic/wp-calypso/pull/10278 to avoid double notices. I think we should rely on `componentWillReceiveProps` to trigger these notices.